db/migrations/create_peep_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps (id SERIAL PRIMARY KEY, content VARCHAR(280), time VARCHAR(60));
```

Comment:
Time could perhaps be added automatically (something like this: "time TIMESTAMP DEFAULT CURRENT_TIMESTAMP;")

Reply:
I did not know you could do this, thats very useful - thanks for the tip!

Reply:
Yeah, or you can simply do time TIMESTAMP